lib/wanda_web/controllers/execution_controller.ex

```diff
@@ -59,4 +75,31 @@ defmodule WandaWeb.ExecutionController do
 
     render(conn, execution: execution)
   end
+
+  def start(
+        conn,
+        _params
+      ) do
+    %{
+      execution_id: execution_id,
+      group_id: group_id,
+      targets: targets,
+      env: env
+    } = Map.get(conn, :body_params)
+
+    case Server.start_execution(execution_id, group_id, Target.from_list(targets), env) do
+      :ok ->
+        conn
+        |> put_status(:accepted)
+        |> json(%{
+          execution_id: execution_id,
+          group_id: group_id
+        })
+
+      {:error, reason} ->
+        conn
```

Comment:
Could we try to use the error view here?

Reply:
bd1ff4bab192c4bfd00d778d10bb605d0c9a4a99 and 44a9eec360b38b02a997c84b527266137f459b95